### ISSUE TITLE
Gulp serve after SCSS file changes not getting refelcted

### ISSUE THREAD

**Issue by rabwill** [SharePoint/sp-dev-docs#5395]
## Category

- [x] Bug

## Describe the bug

Not able to make SCSS file changes and test it in workbench

## Steps to reproduce

1) Edit SCSS file of a react component
2) gulp serve as usual in sharepoint environment , using `?loadSPFX=true&debugManifestsFile=https://localhost:4321/temp/manifests.js`
Also tried this `?debug=true&noredir=true&debugManifestsFile=https://localhost:4321/temp/manifests.js` as suggested when we do a serve.

## Expected behavior

Can see the changes in styles right away

## Developer environment

- OS: Windows 10
- Framework: Node 10.15.3
- Browser(s): Chrome
- Tooling: VS Code, SPFx v1.10.0 (REACT component)

Just saw that the changes have no impact since today, was working fine before
Any help would be appreciated.

**Comment by StfBauer** [comment]
SCSS is unrelated to this. The issue is that when you debug with the query string you will be asked the first time:

![Screenshot 2020-02-28 at 11 14 51](https://user-images.githubusercontent.com/5503835/75540123-bbd2e980-5a1b-11ea-90a9-475fcd142f45.png)

Then you accept and the first load everything is loading normally
![Screenshot_2020-02-28_at_11_20_12](https://user-images.githubusercontent.com/5503835/75540567-8f6b9d00-5a1c-11ea-9952-4857b540f412.png)
.

After you change something in your files the compilation kicks in as is should.

![Screenshot 2020-02-28 at 11 24 02](https://user-images.githubusercontent.com/5503835/75540712-d8bbec80-5a1c-11ea-907d-1cf1de8776d3.png)

On the second load the popup do not show up anymore and only the manifest is loaded but not anything else.

The workaround is to open every time a new browser tab. This issue happens in Firefox too

**Comment by rabwill** [comment]
Yes @StfBauer noticed this in any changes as well and have updated the issue title. Previously the changes were reloaded in the same tab.

**Comment by StfBauer** [comment]
@rabwill There is an intermediate dirty fix to this. 

When you add for development the following line of code:

```js
sessionStorage.clear();
```

Then everything is back to normal and the web part loads as it always has been.

**Comment by lemonpowah** [comment]
This is happening also on SPFx 1.9.1, the clearing the sessionStorage works, thx @StfBauer.
It seems that something broke in SPO and not in SPFx.

**Comment by rabwill** [comment]
This is working now today . I am getting the prompts and changes reflected
Browser : MS edge (chromium)

**Comment by DakotaWray2** [comment]
Dumping sessionStorage is not working for me. Really impacts rapid development when you cannot see changes reflected on the workbench. Seems like this is a persistent (pun intended) issue.

**Comment by sankarkumar23** [comment]
Yes its not working even with sessionStorage.
